src/main/java/seedu/address/model/task/Task.java

```diff
@@ -3,8 +3,9 @@
 import static java.util.Objects.requireNonNull;
 
 public class Task {
-    private final String description;
```

Comment:
I don't think it is a good idea to make it not immutable

Reply:
Agreed. Updated the implementation to ensure immutability of the task. Great suggestion putao!

Reply:
LFG!! :fire::fire::fire:

Reply:
👍 👍 👍 👍 👍 👍 👍